scripts/docker_compose_config.json

```diff
@@ -0,0 +1,65 @@
+{
+    "etna_path": "..",
+    "components": [
+        {
+            "name": "virtual-vehicle-utility",
+            "replace": false,
+            "path": "../../virtual-vehicle",
+            "force_rebuild": false
+        },
+        {
+            "name": "virtual-plc",
+            "replace": false,
+            "path": "../../virtual-plc-arduino-opta",
+            "force_rebuild": false
+        },
+        {
+            "name": "module-gateway",
+            "replace": false,
+            "path": "../../module-gateway",
+            "force_rebuild": false
+        },
+        {
+            "name": "external-server",
+            "replace": false,
+            "path": "../../external-server",
+            "force_rebuild": false
+        },
+        {
+            "name": "http-api",
```

Comment:
What if the name of the component is wrong?

Reply:
Added config validation